config/release.js

```diff
@@ -0,0 +1,18 @@
+/* jshint node:true */
+/* var RSVP = require('rsvp'); */
+var execSync = require('child_process').execSync;
+
+module.exports = {
+  publish: true,
+  afterPublish: function(project, versions) {
+    // Publish dummy app with docs to gh-pages
+    runCommand('ember github-pages:commit --message "Released ' + versions.next + '"');
```

Comment:
I'm not sure I follow what you mean. These are currently being run in order (not at the same time). See the definition of runCommand below (it's using execSync). Have I misunderstood how execSync works?